action.yml

```diff
@@ -14,15 +14,20 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        repository: deephaven/action-assert-dh-tables-present
+        path: action-assert-dh-tables-present
```

Comment:
It seems weird for this change to be needed if this was working before.  I think for this action to be self-consistent, it probably needs to check out the specific version or tag that it is.  But that seems Rube Goldberg, like we're doing something wrong.

Reply:
See here:  https://github.com/n3wt0n/CompositeActionsArchive
I think you need to use `${{ github.action_path }}` and remove the checkout.

Reply:
The action worked locally, it didn't work when I tried to use it on a different project. That change might fix it, I'll give it a shot

Reply:
Worked both in my remote project and here so this is good to go